src/app/component/auth/components/google-btn/google-btn.component.scss

```diff
@@ -0,0 +1,32 @@
+:host {
+  width: 100%;
+}
+
+.google-sign-in {
+  display: flex;
+  padding: 0;
+  height: 40px;
+  width: 100%;
+  background: #fff;
+  border: 0;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.24), 0 0 2px rgba(0, 0, 0, 0.24);
+  border-radius: 3px;
+}
+
+.google-sign {
+  padding: 11px 0 11px 11px;
+  align-self: center;
+}
+
+.google-text-sign-in {
+  flex: 1 1 auto;
+  align-self: center;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 16px;
+  letter-spacing: 0.21875px;
```

Comment:
Why this value so strange?

Reply:
<img width="1052" alt="Screenshot 2020-10-26 at 17 01 06" src="https://user-images.githubusercontent.com/46226026/97189088-e47ad100-17ac-11eb-90c4-989c673794cd.png">
according to design